org.adempiere.base/src/org/compiere/util/CLogFile.java

```diff
@@ -334,6 +337,22 @@ public synchronized void publish (LogRecord record)
 		}
 	}	//	publish
 
+	private String getPrefix() {
+		try
+		{
+			if (DB.isConnected()) {

```

Comment:
need investigation whether this could cause infinite loop when db connection is not available for whatever reason.

Reply:
Note that we have to be very careful with the use of DB connection and MSysConfig inside Log handler since one of the potential exception that's to be logged is exception connection to DB.